frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -31,6 +31,17 @@ <h5 class="fw-light">Challenge Settings</h5>
           <i class="fa fa-trash red-text" aria-hidden="true"></i>
         </a>
         <br />
+        <div class="ev-action">
+          <div *ngIf="isChallengeHost" class="col-sm-1.5 col-xs-1.5 ev-upload-action">
+            <span class="fs-16 fw-light">
+              <strong class="text-light-black">Challenge Overview </strong>
+            </span>
+            <a class="pointer fs-16" (click)="editChallengeOverview()">
+              <i class="fa fa-pencil" aria-hidden="true"> </i>
+            </a>
+          </div>
+        </div>
+        <br />
```

Comment:
please remove this as it's leading to an extra line gap

Reply:
Done!